### PR TITLE
Coinjoin confirmation page redesign

### LINKED ISSUE #7163: Simplify CoinJoin confirmation page
- Remove strategy selection (`Recommended`/`Fast`)
- Remove metrics
- Remove redundant tickbox
- Add reassurance about `time expectation`, `disconnection option`, and `bitcoin doesn't leave Trezor` 

confirmation

This has an impact on `custom setup` but that part is not necessary for coinjoin MVP.

### PR DESCRIPTION
**Pull request by komret**
## Description

Simplification of coinjoin confirmation page:
- removal of pre-defined strategies
- removal of fee calculation
- redesign

Follow-ups:
- custom setup #7431
- remnoval of fast fee rates median #7432

## Related Issue

Resolve #7163

## Screenshots:
Screenshot 2023-01-20 at 17 50 57

### CODE REVIEW

File: packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx [hunk 1]
```diff
@@ -112,123 +146,82 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
...
-                    </StyledCheckbox>
-                    <TooltipSymbol content={<Translation id="TR_DEVICE_CONNECTED_TOOLTIP" />} />
-                </CheckboxWrapper>
+                <TopRow>
+                    <Translation id="TR_COINJOIN_SETUP" />
+                </TopRow>
+                <Tiles>
+                    {tiles.map(tile => (
+                        <Tile
+                            key={tile.description.props.title}
```
Comment (matejkriz) [inline packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx]: isn't that undefined?
image

**Comment by matejkriz** [comment]
[packages/Trezor_T_Top_Black_Connected.png](https://github.com/trezor/trezor-suite/pull/7433#diff-7dd5971d841e8c5d1c092cc76b74bbe4b0828ea2305319d77d536e6c1196339e) image is pushed by mistake, right? @komret 

And the quality of [packages/suite-data/files/images/png/Trezor_T_Connected.png](https://github.com/trezor/trezor-suite/pull/7433#diff-9967f54cadde1e5617f89f49a758d1ad32b7f33ce415108054222edf28a3157c) is too bad, would be nice have sligthly better quality and provide also @2x version

File: packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx [hunk 2]
```diff
@@ -248,13 +241,11 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
...
-            <StyledTooltipButton
-                onClick={anonymize}
-                isDisabled={isDisabled}
-                tooltipContent={getButtonTooltipMessage()}
-            >
-                <Translation id="TR_ANONYMIZE" />
-            </StyledTooltipButton>
+            <Tooltip content={getButtonTooltipMessage()}>
+                <StyledButton onClick={anonymize} isDisabled={isDisabled}>
+                    <Translation id="TR_ANONYMIZE" />
```
Comment (matejkriz) [inline packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx]: By the design, there is a new copy different from the one in BalanceSection ("Start coinjoin" vs "Anonymize") so maybe it makes sense to create a new message id here.

**Comment by matejkriz** [comment]
Except the Tile key, the image of model T and notes for the design/copy,  it looks good to me 👍

File: packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx (hunk 1 above)
Comment (komret) [reply packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx]: 5eca8e7fc665902d1c5124883fa1cefb2034f08f

File: packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx (hunk 2 above)
Comment (komret) [reply packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx]: 5eca8e7fc665902d1c5124883fa1cefb2034f08f

**Comment by komret** [comment]
> [packages/Trezor_T_Top_Black_Connected.png](https://github.com/trezor/trezor-suite/pull/7433#diff-7dd5971d841e8c5d1c092cc76b74bbe4b0828ea2305319d77d536e6c1196339e) image is pushed by mistake, right? @komret

That's right, removed in 5eca8e7fc665902d1c5124883fa1cefb2034f08f.

> And the quality of [packages/suite-data/files/images/png/Trezor_T_Connected.png](https://github.com/trezor/trezor-suite/pull/7433#diff-9967f54cadde1e5617f89f49a758d1ad32b7f33ce415108054222edf28a3157c) is too bad, would be nice have sligthly better quality and provide also @2x version

Also fixed in 5eca8e7fc665902d1c5124883fa1cefb2034f08f.

**Review by matejkriz — APPROVED** [review]
nice, please rebase&merge

**Pull request merged**